profile

```diff
@@ -149,14 +149,15 @@ function update_profile() {
 
 function setup-role() {
   prompt AWS_PROFILE "What should we call this profile [no spaces]? (e.g. ops) " "ops"
-  prompt AWS_ACCOUNT_ID "What is your AWS Account ID? (e.g. 324149397721)" 
+  prompt AWS_ACCOUNT_ID "What is your AWS Account ID? (e.g. 324149397721)"
```

Comment:
remove `prompt AWS_ACCOUNT_ID "What is your AWS Account ID? (e.g. 324149397721)"` it doesn't need anymore

Reply:
We need both prompts.